package.json

```diff
@@ -49,7 +53,10 @@
     "net": "^1.0.2",
     "node-fetch": "^2.6.0",
     "web3": "^1.2.0",
-    "web3-provider-engine": "^15.0.3"
+    "web3-provider-engine": "^15.0.3",
+    "@openzeppelin/gsn-helpers": "^0.1.9",
+    "@openzeppelin/network": "^0.2.7",
+    "react": "^16.9.0"
```

Comment:
[Major] React is a very heavy dependency, I guess you added by accident, please remove it and ensure you also update your package-lock.json

Reply:
Not really... one of the openzeppelin tools (gsn helpers I believe) requires it...

Reply:
In case they require it they should added as subvendor of itself, please remove this